client/lib/route/normalize.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import untrailingslashit from '.';
```

Comment:
I don't think this will work. Perhaps you meant:
```suggestion
import untrailingslashit from './untrailingslashit';
```
?

Reply:
We can import from the specific module like you suggest if you'd prefer that.

Reply:
I'd recommend depending on the sibling module directly, instead, as it would make the dependency graph a bit saner. It's probably still fine, though.